src/formElementPlugins/ApplicationViewWrapper.tsx

```diff
@@ -108,10 +115,10 @@ const ApplicationViewWrapper: React.FC<ApplicationViewWrapperProps> = (props) =>
         })
       if (jsonValue === null)
         // Reset response if cleared
-        await responseMutation({
+        await onSaveUpdateMethod({
           variables: {
             id: currentResponse?.id as number,
-            value: null,
+            value: { text: undefined },
```

Comment:
is this just 'cosmetic' or was there something up with null ?

Reply:
There was a reason -- I think it's because the listBuilder needs the .text and it was causing a crash somewhere when it was getting no text field. This might change though once I refactor to the data shape you described, so will revert this if I can.

Reply:
Have reverted 55d5b51e -- seems to be okay